src/store/modules/nft.js

```diff
@@ -785,6 +793,34 @@ const actions = {
       defaultPaymentCurrency: data.defaultPaymentCurrency,
     });
   },
+  async fetchNFTBookPaymentPriceInfoByClassIdAndPriceIndex(
+    { commit },
+    { classId, priceIndex }
+  ) {
+    const {
+      data: { fiatPrice, LIKEPrice },
+    } = await this.$api.get(
+      api.getNFTBookPaymentPrice({ classId, priceIndex })
+    );
+    const info = { fiatPrice, LIKEPrice };
+    commit(TYPES.NFT_SET_NFT_CLASS_PAYMENT_PRICE_INFO, {
+      classId,
+      priceIndex,
+      info,
+    });
+  },
+  async lazyFetchNFTBookPaymentPriceInfoByClassIdAndPriceIndex(
+    { getters, dispatch },
+    { classId, priceIndex }
+  ) {
+    const info = getters.getNFTClassPaymentPriceById(classId, priceIndex);
+    if (!info) {
+      await dispatch('fetchNFTBookPaymentPriceInfoByClassIdAndPriceIndex', {
+        classId,
+        priceIndex,
+      });
+    }
```

Comment:
Returning `info` for these function might make them more useful in more scenario